x-pack/plugins/alerts/server/alerts_client_factory.test.ts

```diff
@@ -77,7 +78,7 @@ beforeEach(() => {
 
 test('creates an alerts client with proper constructor arguments when security is enabled', async () => {
   const factory = new AlertsClientFactory();
-  factory.initialize({ securityPluginSetup, ...alertsClientFactoryParams });
+  factory.initialize({ securityPluginSetup, securityPluginStart, ...alertsClientFactoryParams });
```

Comment:
__note:__ eventually we'll remove `securityPluginSetup` completely, but we're not there yet.